### DISCUSSION TITLE
What role should water play in our game?

### DISCUSSION THREAD

**Discussion by bzm3r** [Leafwing-Studios/Emergence#77] [General]
We haven't spoken much about it.  We probably should speak more about it?

This made me think of it: https://nickmcd.me/2020/04/15/procedural-hydrology/

**Comment by alice-i-cecile** [comment]
Water is needed for plants to live.

**Comment by alice-i-cecile** [comment]
Water falls from the sky in rain.

**Comment by alice-i-cecile** [comment]
Water forms rivers and streams.

**Comment by alice-i-cecile** [comment]
Animals need to drink water.

**Comment by alice-i-cecile** [comment]
Water is stored in the soil, but can run-off.

**Comment by alice-i-cecile** [comment]
Water can erode soil.

**Comment by bzm3r** [comment]
This makes me think that in our game, water engineering is probably something that starts become more relevant after we have implemented more climate related mechanics. This will allow us to determine regions which end up being dry/wet-er, which will then require the user to do various things in order to transport water/irrigate soil.

**Reply by alice-i-cecile** [reply]
I think you probably want to prototype:

1. Plants need water
2. Water is stored in the soil (start with a default regen rate per tile type)
3. Water can be moved

Then you can start adding more sophisticated "how does water get places" mechanisms :)
